worker-core/pom.xml

```diff
@@ -104,6 +104,11 @@
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-tracking-report</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.workerframework</groupId>
+            <artifactId>worker-message-prioritization-rerouting</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Put runtime-scope dependencies after the compile-scope ones and before the test-scope ones.